cse/SearchEngine.py

```diff
@@ -330,8 +366,10 @@ def processAll(self, tokenTuples):
 
 if __name__ == '__main__':
     se = SearchEngine("data")
-    #se.index()
+    se.index()
```

Comment:
Really every time?

Reply:
PR #54

Reply:
this is only for testing anyway... we have no entry point for the search engine specified yet

it changes from commit to commit 😆